q2_annotate/busco/busco.py

```diff
@@ -21,93 +20,71 @@
     _draw_selectable_summary_histograms
 
 from q2_annotate.busco.utils import (
-    _parse_busco_params, _collect_summaries, _rename_columns,
+    _parse_busco_params,
     _parse_df_columns, _partition_dataframe, _calculate_summary_stats,
-    _get_feature_table, _cleanup_bootstrap, _get_mag_lengths,
-    _validate_lineage_dataset_input, _validate_parameters
+    _get_feature_table, _cleanup_bootstrap,
+    _validate_lineage_dataset_input, _extract_json_data,
+    _validate_parameters
 )
 from q2_annotate._utils import _process_common_input_params, run_command
 from q2_types.per_sample_sequences import MultiMAGSequencesDirFmt
 from q2_annotate.busco.types import BuscoDatabaseDirFmt
 from q2_types.feature_data_mag import MAGSequencesDirFmt
 
 
-def _run_busco(
-    output_dir: str,
-    mags: Union[MultiMAGSequencesDirFmt, MAGSequencesDirFmt],
-    params: List[str]
-) -> Dict[str, str]:
-    """Evaluates mags for all samples using BUSCO.
+def _run_busco(input_dir: str, output_dir: str, sample: str, params: List[str]):
```

Comment:
Can you please rename `sample` to `sample_id` for consistency with the other methods?